web/src/HUD.scss

```diff
@@ -5,6 +5,10 @@
   align-items: stretch;
   width: 100%;
   min-height: 100vh;
+
+  ::selection {
```

Comment:
this doesn't seem right. We don't want to highlight the selection in other panes

Reply:
Oh yeah, I should have explained my thinking here. This adjusts the default word selection color so it's our brand color. I actually thought it was more inconsistent to have the Log Pane use this color, while the rest of the interface has a slightly different (default) blue highlight color. The fact that the two blues were very close, but a little bit off, seemed not very intentional.

That said, this doesn't need to be part of this PR, so I'll revert it for now.